multiqc/report.py

```diff
@@ -795,7 +795,7 @@ def compress_json(data):
     # Stream to an in-memory buffer rather than compressing the big string
     # at once. This saves memory.
     buffer = io.BytesIO()
-    with gzip.open(buffer, "wt", encoding="utf-8", compresslevel=9) as gzip_buffer:
+    with gzip.open(buffer, "wt", encoding="utf-8", compresslevel=6) as gzip_buffer:
```

Comment:
Just prefer constants in code to be justified:

```suggestion
    with gzip.open(buffer, "wt", encoding="utf-8", compresslevel=6) as gzip_buffer:
        # The compression level 6 gives 10% speed gain vs. 2% extra size, in contrast to default compresslevel=9 
```